src/Ledger/Conway/Conformance/Utxo.agda

```diff
@@ -95,7 +95,7 @@ data _⊢_⇀⦇_,UTXO⦈_ : UTxOEnv → UTxOState → Tx → UTxOState → Type
     ∙ L.refScriptsSize utxo tx ≤ pp .maxRefScriptSizePerTx
 
     ∙ ∀[ (_ , txout) ∈ txoutsʰ .proj₁ ]
-        inject ((overhead + L.utxoEntrySize txout) * coinsPerUTxOByte pp) ≤ᵗ getValueʰ txout
+        ((overhead + L.utxoEntrySize txout) * coinsPerUTxOByte pp) ≤ coin (getValueʰ txout)
```

Comment:
This isn't equivalent to the previous line, since that one also ensures that no negative amounts are put in an output.

We either need a `TokenAlgebra` instance with negative amounts (at least being supported) because `mint` must be able to contain them to burn tokens, or we have to do what the implementation does (I think) which is to have a bunch of different types. I much prefer having this inequality over another type.